workspace/src/main/java/Tests/BaseTest.java

```diff
@@ -0,0 +1,26 @@
+package Tests;
+
+import Pages.RegisterPage;
+import org.junit.After;
+import org.junit.Before;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class BaseTest {
+   public static  WebDriver driver = null;
+
+    @Before
+    public void setUp() {
+        String path;
+
+        System.setProperty("webdriver.chrome.driver", "C:\\Users\\user\\Desktop\\chromedriver.exe");
+        driver = new ChromeDriver();
+        path = "http://newtours.demoaut.com/mercuryregister.php?osCsid=300994835431581d933f180c47baac96";

```

Comment:
The path is going directly to the register page and not the home page.
When you do that you are missing the "Register" button to go to the register page and that should be a part of the test.